ja_jp/calendar-manage-calendars.adoc

```diff
@@ -0,0 +1,63 @@
+=== カレンダーの管理
+==== カレンダーを共有する
+アカウントで有効になっていれば、イベント、予定、ミーティングなどを含むカレンダーをほかの人と共有することができます。
+
+. btn:[カレンダー] タブを選択します。
+. 共有希望のカレンダーを右クリックし、 *共有...* をクリックします。
+. *人々をメールで招待（最も柔軟）* をクリックします。
+. *リンクを持っている人なら誰でも* のドロップダウンメニューから適切なアクセス許可を選択します。
+
+表示:: 共有先のユーザーは 非公開イベント *を除いた* すべての予定を閲覧できます。
+
+予定なし/予定ありの時間だけ表示:: 共有先のユーザーは予定の詳細を確認できず、カレンダーに予約が入っているか入っていないかのみ確認できます。
+
+表示（非公開イベントを含む）:: 共有先のユーザーは非公開のイベント *を含め* すべての予定を閲覧できます。
+
+表示および編集:: 共有先のユーザーは非公開のイベント *を除いた* すべての予定の閲覧と編集ができます。
+
+表示、編集、追加、および削除:: 共有先のユーザーは非公開のイベント *を除いた* すべての予定の閲覧と編集ができます。また、あなたのカレンダーに対して、非公開のイベント *を除いて* 、予定の追加と削除を行うことができます。
+
+表示、編集、追加、および削除（非公開イベントを含む）:: 共有先のユーザーは 非公開のイベント *を含め* すべての予定の閲覧と編集ができます。また、あなたのカレンダーに対して、非公開のイベント *を含め* 予定の追加と削除を行うことができます。
+
+. カレンダーを共有する宛先のメールアドレスを入力します。
+. カレンダーを既に共有している場合、 *変更* をクリックすると、共有中ユーザーのアクセス権の変更、または共有の許可を取り下げることができます。
+. btn:[保存] をクリックします。
+. {product-short}は共有先にリンクを送り、メールを受け取った相手は共有を受け入れるか拒否するかを選択できます。
+
+==== 共有カレンダーの招待を承諾する
+誰かがその人のカレンダーをあなたと共有すると、共有を通知するメールが届きます。
+
+. *共有を受け入れる* をクリックして、自分のカレンダーに追加します。
+共有したカレンダーは「その他」に表示されます。
+. *拒否* をクリックして、共有のリクエストを拒否します。
+
+=== カレンダーを追加する
+以下の方法で新しいカレンダーを追加することが可能です。
+
+==== 手動での追加
+. btn:[カレンダー] タブを選択し、*マイカレンダー* にマウスカーソルを合わせます。
+. 右に表示する image:graphics/plus.svg[plus icon, width=20] アイコンをクリックします。
+. カレンダー名を入力します。
+. 他のカレンダーのイベントと区別できるように、表示する色を選択します。
+
+==== カレンダーの購読
+. btn:[カレンダー] タブを選択し、*その他* にマウスカーソルを合わせます。
+. ギアのアイコンをクリックし、*ほかの人のカレンダーを追加* をクリックします。
+. 購読したいカレンダーの持ち主のメールアドレスを入力します。
+. btn:[保存] をクリックします。
+
+NOTE: 相手が事前にカレンダーを共有している場合のみ追加することが可能です。
+
+
+// 実際のUI上では、以下の機能を確認できませんでした。
+// 念のため、直訳します。
+//
+// ==== 休日のカレンダーの追加
+// この機能でご利用中のカレンダーにお住いの国の休日を追加します。
+//
+// . btn:[カレンダー] タブをクリックし、*Holidays* を選択します。
+// . image:graphics/plus.svg[plus icon, width=20] アイコンをクリックします。
+// . カレンダーに追加したい国の休日を選択します。
+// . *次へ* をクリックし、休日カレンダーが表示する色を選択します。
+// . btn:[保存] をクリックします。
+// 新しく参加する休日カレンダーは *Holidays* の直下に表示されます。
```

Comment:
The English text has been also commented out:

>  // Dawood - Commenting this block since the holiday-calendar zimlet is not officially released and we do not plan to release it.
> //==== Add a Holiday Calendar
> //This feature populates your calendar with holidays in your country.
> //
> //. Click *Calendar* and hover over *Holidays*.
> //. Click image:graphics/plus.svg[plus icon, width=20].
> //. Choose the country whose holidays you want to view in your calendar.
> //. Click *Next* and choose the color to differentiate this calendar from the rest.
> //. Click btn:[Save] to save the changes.
> //Your new subscribed calendar appears under *Holidays*.

As long as the English version of the text exists, I would like to keep the Japanese version as well.

Reply:
I got it.